js/bithumb.js

```diff
@@ -349,6 +350,39 @@ module.exports = class bithumb extends Exchange {
         return this.parseTicker (data, market);
     }
 
+    parseOHLCV (ohlcv, market = undefined) {
+        //
+        //     [
+        //         "0.02501786",
+        //         "0.02501786",
+        //         "0.02501786",
+        //         "0.02501786",
+        //         "0.0000",
+        //         1591488000000
+        //     ]
+        //
+        return [
+            this.safeInteger (ohlcv, 0),
```

Comment:
I think the timestamp should be `this.safeInteger (ohlcv, 5)` here, lmk if we're missing something.

Reply:
Oh, i think i didn't changed comment when i copy paste from other exchange;
Code is right, just comment is wrong sorry.